src/components/Global/IframeWrapper/StartVerificationView.tsx

```
@@ -1,6 +1,6 @@
 'use client'
 
-import { ThinkingPeanut } from '@/assets'
+import { PeanutThinking } from '@/assets'
```

ESLint

[error] '`@/assets`' import is restricted from being used. Import from a specific file instead of the '`@/assets`' barrel — barrels force the bundler to load every re-export and hurt build perf. See CLAUDE.md.

(no-restricted-imports)